app/mailers/pageflow/user_mailer.rb

```diff
@@ -11,5 +11,15 @@ def invitation(options)
         mail(to: @user.email, subject: t('.subject'), from: Pageflow.config.mailer_sender)
       end
     end
+
+    def invitation(options)
+      @user = User.find(options['user_id'])
+      @account_id = options['account_id']
+
+      I18n.with_locale(@user.locale) do
+        headers('X-Language' => I18n.locale)
+        mail(to: @user.email, subject: t('.subject'), from: Pageflow.config.mailer_sender)
+      end
```

Comment:
Let's extract a private method for the `with_locale` stuff to reduce duplication.